processing/src/main/java/io/druid/query/NoopQueryRunner.java

```diff
@@ -28,9 +28,19 @@
 */
 public class NoopQueryRunner<T> implements QueryRunner<T>
 {
+  private final Sequence<T> results;
+
+  public NoopQueryRunner() {
+    results = Sequences.empty();
+  }
+
+  public NoopQueryRunner(Sequence<T> results) {
```

Comment:
This isn't very noop-y… it actually does something!

IMO this could be an anonymous query runner defined where you're using it, or a query runner named something like `FixedSequenceQueryRunner` if it needs to have a name.